src/contrib/mlir/compiler/compiler.cpp

```diff
@@ -14,8 +14,8 @@
 // limitations under the License.
 //*****************************************************************************
 
-// NOTE: This file follows nGraph format style and naming convention since it
-// exposes a public API to the rest of nGraph codebase.
+// NOTE: This file follows nGraph format style, and nGraph naming convention for public method names only, 
+// since they are exposed to the rest of nGraph code-base.
```

Comment:
Something is missing here. We should say that we use MLIR format style and naming convention for the rest.